tests/ui/process/win-command-curdir-no-verbatim.rs

```diff
@@ -0,0 +1,35 @@
+// Test that windows `creation_flags` extension to `Command` works.
```

Comment:
this is wrong

Reply:
Oh, lol. I copy/pasted the template and forgot to change the comment...